frontend/src/BenchmarkData.res

```diff
@@ -0,0 +1,52 @@
+open Belt
+
+module PullNumberId = Id.MakeComparable({
+  type t = option<int>
+  let cmp = (a, b) => Pervasives.compare(a, b)
+})
+
+type timeseries = array<array<float>>
+type byMetricName = Map.String.t<(timeseries, array<{"commit": string, "runAt": Js.Date.t}>)>
+type byTestName = Map.String.t<byMetricName>
+type byPullNumber = Map.t<PullNumberId.t, byTestName, PullNumberId.identity>
+type t = byPullNumber
```

Comment:
For my own curiosity, what's the reason for creating this `type t`?

Reply:
Just following the convention to have a top-level "main" type for the module. The benchmark data is represented as `BenchmarkData.t`. The `byPullNumber` is just a bit more explicit in this context.

If I were to add an interface to this file, I wouldn't export `byPullNumber`.